src/mbgl/programs/symbol_program.hpp

```diff
@@ -94,7 +94,7 @@ class SymbolIconProgram : public Program<
                                        const TransformState&);
 };
 
-class SymbolSDFProgram : public Program<
+class SymbolSDFIconProgram : public Program<
```

Comment:
I initially tried to just parameterize `SymbolSDFProgram` on the paint property types, but it was proving sort of unweildy.   Opted to duplicate the declarations and minimize code duplication in implementation via the shared  `makeSDFValues` function